### ISSUE TITLE
"Add Option" tooltip is always out of view

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2287]
### Expected behavior
Tooltip should be in the viewport

### Actual Behavior
It never is
![dumb_product_and_cart_completed](https://cloud.githubusercontent.com/assets/72819/26132537/9e7a699a-3ad3-11e7-9762-6a928526ecf8.png)

### Steps to Reproduce the Behavior
1. Create a product
1. Mouse over the "plus" icon
1. Observe tooltip is partially out of view

### Versions
```
Node: 7.10.0
NPM: 4.2.0
Meteor Node: 4.8.2
Meteor NPM: 4.5.0
Reaction CLI: 0.8.2
Reaction: 1.2.0
Reaction branch: release-1.2.0
Docker: 17.03.1-ce
```

**Comment by kieckhafer** [comment]
@zenweasel will be fixed via PR #2642 

![screen_shot_2017-08-04_at_3_03_29_pm__2_](https://user-images.githubusercontent.com/4482263/28988749-4704b986-7926-11e7-92c6-fbfaf9ee1d11.png)

**Comment by kieckhafer** [comment]
Fixed in #2642
